docs/sources/review/lint-prose/rules.md

```diff
@@ -0,0 +1,676 @@
+---
+date: "2024-06-25"
+description: A description of every Grafana Labs prose linting rule.
+menuTitle: Rules
+review_date: "2024-06-25"
+title: Vale rules
+---
+
+# Vale rules
+
+<!-- These are our style rules. -->
+<!-- vale Grafana.We = NO -->
+
+Vale codifies our style guide into a series of rules that can be checked against your prose.
+The following is a list of all the rules that we have defined.
```

Comment:
Since you also like being a nitpicker except in real time while I'm trying to write a document. Troll.
```suggestion
The following is a list of all the rules that we've defined.
```